### DISCUSSION TITLE
[oauth] Keycloak provider support

### DISCUSSION THREAD

**Discussion by 9teen90nine** [langfuse/langfuse#1851] [Ideas]
### Describe the feature or potential improvement

In self-hosted environments, keycloak is a #1 option to have an oAuth bridge. It is already supported in next-auth. https://next-auth.js.org/providers/keycloak

**Comment by marcklingen** [comment]
open for PR on this

**Reply by marcklingen** [reply]
Released: https://langfuse.com/changelog/2024-12-03-new-sso-providers-github-enterprise-and-keycloak
